themes/root/templates/RecordDriver/AbstractBase/export-ris.phtml

```diff
@@ -149,12 +164,47 @@ if (is_array($notes)) {
     }
 }
 
+echo 'L4  - ' . $this->serverUrl($this->recordLinker()->getUrl($this->driver)) . "\r\n";
+
 foreach ($this->record($this->driver)->getUrlList() as $url) {
     echo 'UR  - ' . "$url\r\n";
 }
 
 if ($doi = $this->driver->tryMethod('getCleanDOI')) {
-    echo "DO  - $doi\r\n";
+    echo 'DO  - ' . strip_tags($doi) . "\r\n";
+}
+
+$abstract = $this->driver->tryMethod('getAbstractAndSummaryNotes');
+if (is_array($abstract)) {
+    foreach ($abstract as $note) {
+        echo 'AB  - ' . "$note\r\n";
+    }
+} else {
+    $abstract = $this->driver->tryMethod('getItemsAbstract');
```

Comment:
Maybe it would be better to add wrapper methods to the EDS driver to make the method names consistent instead of creating special alternate cases here.

Reply:
OK